components/PurpleButton.tsx

```diff
@@ -0,0 +1,41 @@
+// this is the general layout for the button that allows users to continue
+import React from "react";
+import { StyleSheet, TouchableOpacity, Text, View } from "react-native";
+
+export default function GreyButton({ text, onPress }) {
```

Comment:
we need to be able to dim the button when the user has not filled out all the required info. Are you thinking about creating another component or extending the button here to accept a parameter for this?